src/server/fetch/certs.js

```diff
@@ -0,0 +1,35 @@
+import { promises as fs } from 'fs';
+
+const load = async path => {
+  let cert;
+  try {
+    cert = await fs.readFile(path, 'UTF-8');
+    return cert;
+  } catch (error) {
+    return '';
```

Comment:
I don't think we should suppress the error in production, at very least we should log it out so we can debug any issues